packaging/obs/grafana-ha-cluster-dashboards/grafana-ha-cluster-dashboards.spec

```
@@ -23,15 +23,20 @@ Group:          System/Monitoring
 Url:            https://github.com/ClusterLabs/ha_cluster_exporter
 Source:         %{name}-%{version}.tar.gz
 BuildArch:      noarch
-Requires:       grafana
-BuildRequires:  grafana
+BuildRequires:  shadow
```

According to packaging guidelines, `Require(pre)` is more correct.
```suggestion
Require(pre):   shadow
```